README.md

```diff
@@ -239,12 +239,12 @@ configured using the special *venafi-policy* path which InfoSec teams can use to
     
 ## Quickstart: Enabling Venafi Visibility
 
-# !! Need to rewrite this section. Visibility is on on the policy level now
-1. Visibiliy is enabled at the [PKI role](https://www.vaultproject.io/docs/secrets/pki/index.html) by enabling the `venafi_import` option:
+1. Visibiliy is enabled at the policy level by enabling the `venafi_import` option. The roles that
```

Comment:
The first word has typo.

I noticed two more typos:
- Line 156 search for "enfrcement"
- Line 518 search for "supportiong"

Reply:
Thanks @achuchev! Typos corrected and pushed to branch.